### ISSUE TITLE
could not install emacs within osrf/ubuntu_armhf

### ISSUE THREAD

**Issue by k-okada** [osrf/ubuntu_armhf-docker#1]
TL;DR;

please re-run build-image.sh on 16.04 machine and update docker hub image.

==

I have trouble on installing emacs wihtin osrf/ubuntu_armhf

I can conform this problem not only http://build.ros.org/job/Kbin_uxhf_uXhf__multi_map_server__ubuntu_xenial_armhf__binary/50/consoleFull, but also local docker machine.
after running docker with `$ docker run -m 4g --rm -i -t osrf/ubuntu_armhf:xenial /bin/bash -i` and 
```
root@11d02b9d60c5:/# apt-get -y update && apt-get -y insatll emacs
```
end with
```
Install emacsen-common for emacs24
emacsen-common: Handling install of emacsen flavor emacs24
qemu: uncaught target signal 11 (Segmentation fault) - core dumped
Segmentation fault (core dumped)
ERROR: install script from emacsen-common package failed
dpkg: error processing package emacs24 (--configure):
 subprocess installed post-installation script returned error exit status 1
```

However, it is interesting that if we use https://hub.docker.com/r/snapcraft/xenial-armhf/~/dockerfile/, we do not have this problem. the difference is they use
 1) https://github.com/chihchun/snapcraft-docker/blob/master/baseimage/build-scrtch.sh#L6 or the image.
 2) version 2.5 of qemu-user-static
```
$ docker run --rm -i -t osrf/ubuntu_armhf:xenial /usr/bin/qemu-arm-static --help
usage: qemu-arm [options] program [arguments...]
....
$ docker run  --rm -i -t snapcraft/xenial-armhf:latest /usr/bin/qemu-arm-static --versionqemu-arm version 2.5.0 (Debian 1:2.5+dfsg-5ubuntu10.4), Copyright (c) 2003-2008 Fabrice Bellard
```
So, I ran build-image.sh from 16.04 host machine, which has version 2.5 of qemu-user-static, then we can install emacs within armhf docker.

**Comment by tfoote** [comment]
We've moved over to using https://github.com/osrf/multiarch-docker-image-generation and have regenerated since this request. https://github.com/osrf/multiarch-docker-image-generation/issues/5
